### PR TITLE
Add Qiskit 1.3 release summary to Latest Updates

### LINKED ISSUE #2415: Add Qiskit 1.3 release summary to Latest Updates

### PR DESCRIPTION
**Pull request by abbycross**
Closes #2415

### CODE REVIEW

**Review by Eric-Arellano — APPROVED** [review]

File: docs/guides/latest-updates.mdx [hunk 1]
```diff
@@ -14,24 +14,22 @@ Keep up with the latest and greatest from Qiskit and IBM Quantum&trade;! Gathere
...
-- [`BackendV1`](/api/qiskit/qiskit.providers.BackendV1) is now deprecated in favor of [`BackendV2`](/api/qiskit/qiskit.providers.BackendV2), including the deprecation of `Qobj`, `assemble`, and other scaffolding related to the old V1.
-- Support for Python 3.8 will be dropped in the next release, v1.3. Python 3.9 will then be the new minimal required version.
+- The circuit library is now faster and allows for better high-level optimization, in particular with context-aware updates to `HighLevelSynthesis`.
 
-Remember, because this a minor release, it will not introduce breaking changes for users moving from Qiskit SDK v1.0. Any deprecations announced with this release refer only to things that will now start throwing deprecation warnings. None of these will be removed until the release of Qiskit v2.0, pursuant to Qiskit’s official [deprecation policy](https://github.com/Qiskit/qiskit/blob/main/DEPRECATION.md#).
+- Python 3.13 is now officially supported. Python 3.9 is now the *minimal* supported version.
```
Comment (mtreinish) [inline docs/guides/latest-updates.mdx:30]: Should this be?

```suggestion
- Python 3.13 is now officially supported. Python 3.9 is now the *minimum* supported version.
```

File: docs/guides/latest-updates.mdx [hunk 2]
```diff
@@ -14,24 +14,22 @@ Keep up with the latest and greatest from Qiskit and IBM Quantum&trade;! Gathere
...
+We improved performance by porting additional key components to Rust, including:
+- Core data structures, like `DAGCircuit` or `Target`
+- Transpiler passes, including `BasisTranslator`, `CommutationAnalysis`, or `UnitarySynthesis`, making Qiskit around six times faster compared to v1.2.4
+- Circuit library and synthesis routines, such as `PauliEvolutionGate` or `n_local`
 
-Important deprecations:
-- Primitive V1 reference implementations (like `qiskit.primitives.Estimator`) and tooling are deprecated in favor of [V2 primitives](/api/qiskit/primitives). The primitive V1 base classes remain available.
-- [`BackendV1`](/api/qiskit/qiskit.providers.BackendV1) is now deprecated in favor of [`BackendV2`](/api/qiskit/qiskit.providers.BackendV2), including the deprecation of `Qobj`, `assemble`, and other scaffolding related to the old V1.
-- Support for Python 3.8 will be dropped in the next release, v1.3. Python 3.9 will then be the new minimal required version.
+The circuit library is now faster and allows for better high-level optimization, in particular with context-aware updates to `HighLevelSynthesis`.
```
[the remark this answers is not in the document]
Comment (Cryoris) [reply docs/guides/latest-updates.mdx]: Looks great, thank you!

File: docs/guides/latest-updates.mdx (hunk 1 above)
Comment (abbycross) [reply docs/guides/latest-updates.mdx:30]: I see both on the internet but I think I like minimum better!

File: docs/guides/latest-updates.mdx [hunk 3]
```diff
@@ -5,7 +5,7 @@ description: The latest updates from Qiskit and IBM Quantum, including the lates
 
 # Latest updates
 
-*Last updated 14 November 2024*
+*Last updated 2 December 2024*
```
Comment (abbycross) [inline docs/guides/latest-updates.mdx:8]:
```suggestion
*Last updated 4 December 2024*
```

File: docs/guides/latest-updates.mdx [hunk 4]
```diff
@@ -14,24 +14,22 @@ Keep up with the latest and greatest from Qiskit and IBM Quantum&trade;! Gathere
...
-- Support for Python 3.8 will be dropped in the next release, v1.3. Python 3.9 will then be the new minimal required version.
+- The circuit library is now faster and allows for better high-level optimization, in particular with context-aware updates to `HighLevelSynthesis`.
 
-Remember, because this a minor release, it will not introduce breaking changes for users moving from Qiskit SDK v1.0. Any deprecations announced with this release refer only to things that will now start throwing deprecation warnings. None of these will be removed until the release of Qiskit v2.0, pursuant to Qiskit’s official [deprecation policy](https://github.com/Qiskit/qiskit/blob/main/DEPRECATION.md#).
+- Python 3.13 is now officially supported. Python 3.9 is now the *minimum* supported version.
 
-For more details, read the post about the release on the [IBM Quantum blog](https://www.ibm.com/quantum/blog/qiskit-1-2-release-summary), and explore the [v1.2 release notes](/api/qiskit/release-notes/1.2).
+Explore the [v1.3 release notes](/api/qiskit/release-notes/1.3).
```
Comment (abbycross) [inline docs/guides/latest-updates.mdx:32]:
```suggestion
Explore the [v1.3 release notes](/api/qiskit/release-notes/1.3), and read about this release on the [IBM Quantum blog](https://www.ibm.com/quantum/blog/qiskit-1-3-release-summary).
```

**Pull request merged**